packages/itwin/grouping-mapping-widget/src/api/GroupQueryBuilderApi.ts

```diff
@@ -14,11 +15,6 @@ import {
 } from "@itwin/presentation-components";
 import type { IModelConnection } from "@itwin/core-frontend";
 
-export interface PropertyProps {
-  keys: KeySet;
-  imodel?: IModelConnection;
-}
-
 /* This class demonstrates the key APIs needed to access formatted property information
    suitable to present to end users. */
 export class GroupQueryBuilderApi {
```

Comment:
Where is the `createPropertyDataProvider` static method used? If it is only used in the `PropertyGridWrapper.tsx`, then it makes no sense to have this class.

Reply:
Yes, it was used only here, removed the class. `createPropertyDataProvider` trasferred to `PropertyGridWrapper.tsx`